### ISSUE TITLE
F4, F5, F6, F7 keys mapped to same OF_KEY as LEFT, UP, RIGHT, DOWN

### ISSUE THREAD

**Issue by prisonerjohn** [openframeworks/openFrameworks#5948]
If we look at [ofEvents.h](https://github.com/openframeworks/openFrameworks/blob/master/libs/openFrameworks/events/ofEvents.h#L69-L90), the result of the bitwise OR for the F4, F5, F6, F7, keys is identical to the arrow keys, which makes it impossible to differentiate them in a key event handler.

For example, if we look at F5:
```
   3808 = 1110 1110 0000
OR    5 = 0000 0000 0101
------------------------
   3813 = 1110 1110 0101
```
and UP
```
   3808 = 1110 1110 0000
OR  101 = 0000 0110 0101
------------------------
   3813 = 1110 1110 0101
```
we end up with the same value for `OF_KEY_F5` and `OF_KEY_UP`!

Not sure what the solution is here, maybe change the value of `OF_KEY_MODIFIER` or the bitwise OR operator. 
Or maybe since we're not using GLUT anymore we can just use the [GLFW key macros](http://www.glfw.org/docs/latest/group__keys.html) directly?

**Comment by arturoc** [comment]
the problem with the glfw macros is that some of them clash with characters. The main problem is we use the actual character to check for a key like:

```
if (key == 'a'){
...
}
```
we need to find numbers that are not used for those keys, probably we don't need to use OF_KEY_MODIFIER but we need to figure out codes that are empty in the unicode spec or that are before the symbols you can type.

**Comment by prisonerjohn** [comment]
It looks like there's a nice chunk of 48 unused codepoints from 0870 - 089F. See https://unicode-table.com/en/#syriac-supplement

If it's just a matter of replacing those `OF_KEY_xxx` values, I can go ahead and do that and submit a PR if it appears to work correctly.

**Comment by prisonerjohn** [comment]
Fixed in PR #5956
